CMakeLists.txt

```diff
@@ -182,6 +182,13 @@ endif()
 
 #-------------------------
 # Setup the basic C compiler
+# Some compilation flags are not supported in clang, lets allow users to know
+# whether gcc or clang is used.
+if ("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang")
```

Comment:
I know that I reviewed it, but unfortunately, this is not enough, you need to do a feature test.
It will allow us to run and check everything with new clang once they will support this -fno-var-tracking-assignments without changing rdma-core at all. Otherwise, we won't be able to open this check, because it will never work on "old" platforms.